### DISCUSSION TITLE
Static site generation with vike (formerly vite-ssr-plugin) causes SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by liborjelinek** [primefaces/community#143] [PrimeVue]
Has anybody already integrate PrimeVue with Vike for SSG? My app works works great on "npm run dev", but for "npm run build" it causes `SyntaxError: Cannot use import statement outside a module`.

For example, when I try to generate page containing Button:

```
...
vike v0.4.142 pre-rendering HTML...
(node:65063) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/Users/matt/git-dm/www/.my/blank-vue/node_modules/primevue/button/button.esm.js:1
import Badge from 'primevue/badge';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

**Comment by tugcekucukoglu** [comment]
Could you try to add these lines that official examples of vite-ssg recommends? https://github.com/antfu/vite-ssg/blob/main/examples/single-page/vite.config.ts#L10

`vite.config.js`

```
ssgOptions: {
    script: 'async',
    formatting: 'prettify',
}
```

**Reply by Uzeyir9695** [reply]
I am using `Laravel 10, Vue 3 and Vite`. Same problem with me when I run `php artisan inertia:start-ssr`
error:
```
error:
import Badge from 'primevue/badge';
^^^^^^

SyntaxError: Cannot use import statement outside a module

```
